### ISSUE TITLE
Multiline template variable query expression editor field

### ISSUE THREAD

**Issue by amotl** [grafana/grafana#15216]
Hi there,

thanks for conceiving and maintaining this amazing graphing toolkit.

Within the [Hiveeyes Project](https://hiveeyes.org/), we recently got into adding PostGIS to our stack. This works pretty well, you might want to have a look at the [LDI #6 » GIS distance by threshold
](https://weather.hiveeyes.org/grafana/d/EWFuSqlmz/ldi-6-gis-distance-by-threshold) dashboard where we are currently prototyping this.

### Problem
However, we are currently wrestling with larger SQL query expressions like:
```
WITH reference AS (
  SELECT geopoint AS position FROM ldi_stations WHERE station_id IN ($ldi_station_id_reference)
)
SELECT    station_id
FROM      ldi_stations, reference
ORDER BY  geopoint <-> reference.position
LIMIT     $max_stations;
```

inside the little `<input>` form field of the template variable editor:

![image](https://user-images.githubusercontent.com/453543/52195247-6c8daa00-2857-11e9-8c78-bace1598ea37.png)

### Suggestion
Would converting the query expression field into a multiline `<textarea>` element make sense to you?

Thanks in advance and with kind regards,
Andreas.

**Comment by wetterfrosch** [comment]
+1 on this, especially when people are more and more starting to use IFQL ("Flux"), this one-line-field will be painful.

It would be really cool  to see this in early Grafana 6 already, or maybe even backported to another 5.x release?

**Comment by hugohaggmark** [comment]
This issue was fixed by #15216. Closing this issue as we're trying to limit open GitHub issues in order to better track planned work and features.

Thank You to you for taking the time to create this issue!

**Comment by amotl** [comment]
Dear Hugo,

> Closing this issue as we're trying to limit open GitHub issues.

Sure, good luck!

> This issue was fixed by #15216.

Thanks! Did you mean to reference a different issue / pull request here? This currently points back to itself.

With kind regards,
Andreas.

**Comment by hugohaggmark** [comment]
> Dear Hugo,
> 
> > Closing this issue as we're trying to limit open GitHub issues.
> 
> Sure, good luck!

Thank you @amotl 

> > This issue was fixed by #15216.
> 
> Thanks! Did you mean to reference a different issue / pull request here? This currently points back to itself.
> 
> With kind regards,
> Andreas.

My bad, I hope the new referenced issue is correct 🙈
